updating change log for new release (#257)
infinum/eightshift-libs: CHANGELOG.md

This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a CHANGELOG](https://keepachangelog.com/).

## [4.1.0] - 2022-02-02

### Added
- Eightshift-forms as default allow block types.
- New WP-CLI command to create custom WP-CLI command. Example: `wp boilerplate create_cli_command`.

> - New prop to outputCssVariables called $customSelector.

Reviewer: ```suggestion
- New attribute to `outputCssVariables` called `$customSelector`.
```

> - DI container caching on production, or staging.

Reviewer 2: ```suggestion
- DI container caching on production or staging.
```

- Option to remove default paragraph block placeholder from the frontend if the content is empty.
- Custom post type revisions as default.

> - Limit enqueue admin only to admin area and not block editor.

Reviewer 2: ```suggestion
- Limit enqueue admin only to admin area but not to the block editor.
```

> - Limit enqueue admin only to admin area but not to the block editor.

Reviewer: This should go under `Changed` section of the changelog

Author: done

> - Option ot override default block class prefix in the blocks global settings manifest.

Reviewer 3: ```suggestion
- Option to override default block class prefix in the blocks global settings manifest.
```

> - More tests.

Reviewer: I don't think we need to mention we added more tests, that should be a given 😄

Author: We sound smart :D

Author: removed

### Fixed

> - Changed directory separator from `\` to `DIRECTORY_SEPARATOR` constant to be able to work cross-platform.

Reviewer 4: Previous separator is wrongly listed here, see PR #234
```suggestion
- Changed directory separator from `/` to `DIRECTORY_SEPARATOR` constant to be able to work cross-platform.
```

- Various fixes and improvements.

> 

Reviewer: Can you add the missing releases? I've added them here, so just copy them from here: https://github.com/infinum/eightshift-libs/blob/3.1.0/CHANGELOG.md (just make sure to link them, I just saw that I didn't link them 🙈)

Author: Done
